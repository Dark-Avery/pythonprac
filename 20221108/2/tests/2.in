asdfasdsad
1,2,3,4,5,6,1
(1,1), (2,2), (5,11)
(1,1), (5,5), (80,80)

