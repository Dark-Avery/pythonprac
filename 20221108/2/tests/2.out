Invalid input
Invalid input
3.00
Not a triangle
